crates/services/txpool_v2/src/tests/tests_pending_pool.rs

```diff
@@ -0,0 +1,132 @@
+use fuel_core_services::Service;
+use fuel_core_types::{
+    fuel_tx::{
+        UniqueIdentifier,
+        UtxoId,
+    },
+    services::txpool::TransactionStatus,
+};
+use futures::StreamExt;
+
+use crate::{
+    tests::universe::TestPoolUniverse,
+    TxStatusMessage,
+};
+
+#[tokio::test]
+async fn test_tx_keep_missing_input() {
```

Comment:
So, this should pass when the tx containing the dependent input for the pending tx is submitted? Maybe we include that in the test name.

Reply:
Done.